### ISSUE TITLE
Should affine be set as False in your ADD layers? 

### ISSUE THREAD

**Issue by rshaojimmy** [mindslab-ai/faceshifter#18]
Hi, Thanks for your great implementation!

In the original paper, batch normalization is conducted without affine parameters (they are replaced with attributes and identities modulation parameters).

So in this way, we should explicitly set the flag of affine of BN in the ADD layers as false as follows,

self.BNorm = nn.BatchNorm2d(h_inchannel, affine=False)

Thanks for your response.

**Comment by usingcolor** [comment]
I think you are right.
I only tried with my code, so I don't know how it affects to the training.
